packages/blade/.storybook/react/preview.tsx

```diff
@@ -27,6 +28,19 @@ export const parameters = {
       disable: true,
     },
   },
+  viewport: {
+    viewports: {
+      ...MINIMAL_VIEWPORTS,
+      iPhone6: {
+        name: 'iPhone 6',
+        styles: {
+          height: '667px',
+          width: '375px',
+        },
+        type: 'mobile',
+      },
```

Comment:
Does the MINIMAL_VIEWPORTS doesn't have any mobile viewport by default?

Reply:
they have weird viewports like iPhone-5 or smaller so i added standard one